docs/Gemfile

```diff
@@ -10,7 +10,7 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "~> 3.8.7"
+
 # This is the default theme for new Jekyll sites. You may change this to anything you like.
 gem "minima", "~> 2.5"
 # If you want to use GitHub Pages, remove the "gem "jekyll"" above and
```

Comment:
minor nit, but perhaps to comment could be tweaked slightly as it now referrers to a non-existent line. :)

Reply:
I think I'll can perhaps slightly change the comment after merging with another commit. :)